<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FootDialog</class>
 <widget class="QDialog" name="FootDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="footGV">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>329</width>
     <height>524</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>151</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Datos:</string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="footTypeLbl">
      <property name="text">
       <string>Tipo de pie:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="footTypeVal">
      <property name="text">
       <string>#</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="percentLbl">
      <property name="text">
       <string>Porcentaje:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="percentVal">
      <property name="text">
       <string>#</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGraphicsView" name="thermalScaleGV">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>31</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="angleLbl">
      <property name="text">
       <string>Ángulo:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="angleVal">
      <property name="text">
       <string>180º</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="angleSlider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>540</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-180</number>
   </property>
   <property name="maximum">
    <number>180</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
